<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GVi_Principal</class>
 <widget class="QMainWindow" name="GVi_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GVi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="edRuta">
        <property name="placeholderText">
         <string>Ruta a imagen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="barAvance">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lsPlacas"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnAbrir">
        <property name="text">
         <string>Abrir...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnProcesar">
        <property name="text">
         <string>Procesar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
